server/graphql/common/update.ts

```diff
@@ -22,6 +23,8 @@ export async function createUpdate(_, args, req) {
     throw new Forbidden('Only root users can create changelog updates.');
   }
 
+  await twoFactorAuthLib.enforceForAccountAdmins(req, collective, { neverAskForToken: true });
```

Comment:
The use of `neverAskForToken` is not clear imo. The flag is used because we only want to check if the user has used 2fa to login if the account requires it, I suggest we do this differently.

We could use the twoFactorAuthLib on the login with 2fa flow and there set a session duration that equals the JWT duration. By default the 2fa enforcing will be using the login session. If we need a different session time or rule we can then explicitly set it in the appropriate mutations.

Reply:
I agree that it's not ideal, but I think the solution you proposed would be better if coupled with a concept of "protection level" (soft, hard, etc) as proposed in https://github.com/opencollective/opencollective/issues/6085; which will further complexify this PR that is already quite dense.

As a clarification, I think we can rename `neverAskForToken` to `onlyAskOnLogin`; it might not be the best API but it will make it clearer for other devs reading the code while keeping the door open for further refactoring & clarification when we'll have better specs.